shared/core/index.ts

```diff
@@ -1,6 +1,6 @@
 export * from "./src/HousingCounselors"
 export * from "./src/ami_charts"
 export * from "./src/general"
```

Comment:
I realize that being able to do this is more-or-less the whole point of the exercise here, but exporting across package boundaries like this doesn't feel right to me. If everyone is good with the path in this PR generally, I almost wonder if we shouldn't just retire `shared/core` as a separate directory, and then publish the interfaces out of `backend/core` directly (taking care not to publish the whole backend for size reasons). I'm not sure what the existing best practices / prior art is here, so open to suggestions...

Reply:
@bk3c I think whether the interfaces are in shared/core or backend/core, the entire package gets installed locally to node_modules to the purpose of being accessible by the Next app's Webpack—however, virtually nothing gets imported into the app bundle due to the TypeORM shim we use. So I don't necessarily feel we need to change this approach (we might, and still do, have interfaces that aren't DB-related that we can export to anywhere—thus shared/core still seems relevant).

Reply:
Yeah, ok. We can revisit this in a few months and see if the pattern still makes sense once we've built out more.